resources/intent/fokoos_base/abs/fokoos_abs_nozzle_0.40_layer_0.10_visual.inst.cfg

```diff
@@ -0,0 +1,54 @@
+[general]
+version = 4
+name = Visual
```

Comment:
Uh, isn't that weird? Your profile is called "Visual" but it's type is "fast"?

Reply:
I have changed those profiles name and it's type